### PR TITLE
use astyle code formatter for consistency and style

### LINKED ISSUE #177: style cleanup
the code would be better readable and feel more consistent with cleanups like:

- use "a, b" - not "a,b"
- use "a operator b" - not "aoperator b" nor "a operatorb" nor "aoperatorb"

is there some C style checker we could adopt and integrate via travis-ci?

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
astyle is a configurable automatic C/C++ code formatter.

i did the cleanup step-by-step:
- first config change
- then automatic formatting
- then manually fix stuff that didn't look nice
- repeat

fixes #177.

### CODE REVIEW

File: multigeiger/multigeiger.ino [hunk 1]
```diff
@@ -943,83 +943,81 @@ void sendData2http(const char* host, int sendwhat, unsigned int hvpulses, unsign
...
+    "<body>"
+    "<h1>Configuration</h1>"
+    "<p>"
     "Go to the <a href='config'>configure page</a> to change settings or update firmware."
-  "</p>"
-"</body>"
-"</html>\n";
+    "</p>"
+    "</body>"
+    "</html>\n";
```
Comment (ThomasWaldmann) [inline multigeiger/multigeiger.ino]: automatted formatting kills our string-storage space-saving indentation here. but guess we can live with that.

File: .astylerc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+suffix=none
+recursive
+style=attach
+lineend=linux
+convert-tabs
+indent=spaces=2
```
Comment (ThomasWaldmann) [inline .astylerc]: we could also switch to some different indentation, if we like.

File: .astylerc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+indent-preproc-cond
+pad-comma
+pad-oper
+pad-header
```
Comment (ThomasWaldmann) [inline .astylerc]: these help a lot with readability.

**Comment by rexfue** [comment]
Indent with 2 spaces is OK, style of braces also.
Shouldn't we remove the spaces in the variable definitions?
Regardless of this, it ca be merged.

**Comment by ThomasWaldmann** [comment]
You mean rather use `int a=1;` than `int a = 1;`?

The 2nd one is better readable IMHO.

**Comment by rexfue** [comment]
no no, around line 220

File: multigeiger/multigeiger.ino [hunk 4]
```diff
@@ -201,59 +202,59 @@ const unsigned long GMC_dead_time = 190;  // Dead Time of the Geiger Counter. Ha
...
-
-         unsigned int  GMC_counts             = 0;
-         unsigned int  GMC_counts_2send       = 0;
-         unsigned int  accumulated_GMC_counts = 0;
-         unsigned long count_timestamp        = millis();
-         unsigned long count_timestamp_2send  = millis();
-         unsigned long last_count_timestamp   = millis();
-         unsigned long last_count_timestamp_2send = millis();
-         unsigned long accumulated_time       = 0;
-         unsigned int  last_GMC_counts        = 0;
```
Comment (ThomasWaldmann) [inline multigeiger/multigeiger.ino]: so we talk about this change?

IMHO:
- aligning it that way is quite some work and likely, some change (e.g. a new long variable name) might need changes in multiple lines just to get it all aligned again.
- so it is easier just not to align.

Of course, the same problem exists with alignment of end-of-line comments.

File: multigeiger/multigeiger.ino [hunk 5]
```diff
@@ -201,59 +202,59 @@ const unsigned long GMC_dead_time = 190;  // Dead Time of the Geiger Counter. Ha
...
+unsigned int  GMC_counts_2send = 0;
+unsigned int  accumulated_GMC_counts = 0;
+unsigned long count_timestamp = millis();
+unsigned long count_timestamp_2send = millis();
+unsigned long last_count_timestamp = millis();
+unsigned long last_count_timestamp_2send = millis();
+unsigned long accumulated_time = 0;
+unsigned int  last_GMC_counts = 0;
+unsigned int  hvpulsecnt2send = 0;
+float         Count_Rate = 0.0;
```
Comment (ThomasWaldmann) [inline multigeiger/multigeiger.ino]: this is still half-aligned, so not totally consequent.

**Comment by rexfue** [comment]
I think we misunderstand each other : i don't want the alignment . I want ONE space on each side of the '=', that's enough.
I downloaded your PR, but there is still the alignment, therefore I mentioned it.

**Comment by ThomasWaldmann** [comment]
Line 220 has 1 space on each side of the `=`. Can you just comment at the place where you see an issue? You can go to "Files changed" tab here and just click on a problematic line and then comment there.

File: multigeiger/multigeiger.ino (hunk 5 above)
Comment (rexfue) [inline multigeiger/multigeiger.ino]: ok, my fault, sorry.  Not at the '=', but after the data type (float). That's what  I meant.

File: multigeiger/multigeiger.ino [hunk 6]
```diff
@@ -823,47 +825,45 @@ void displayStatusLine(String txt) {
...
-  jbTone(1174659*freq_factor,    4*time_factor, 1); // D
-  jbTone( 987767*freq_factor,    2*time_factor, 1); // H
-  jbTone(1046502*freq_factor,    2*time_factor, 1); // C
-  jbTone( 987767*freq_factor,    4*time_factor, 1); // H
-  jbTone( 987767*freq_factor,    4*time_factor, 0); // H
-
-  return;
+  int time_factor = 85;
+
+  jbTone(1174659 * freq_factor,    2 * time_factor, 1); // D
```
Comment (rexfue) [inline multigeiger/multigeiger.ino]: Also here: unnecessary alignement

File: multigeiger/multigeiger.ino (hunk 1 above)
Comment (rexfue) [reply multigeiger/multigeiger.ino]: That's OK

File: multigeiger/multigeiger.ino (hunk 5 above)
Comment (ThomasWaldmann) [reply multigeiger/multigeiger.ino]: ok, I'll fix that.

File: multigeiger/multigeiger.ino (hunk 6 above)
Comment (ThomasWaldmann) [reply multigeiger/multigeiger.ino]: ok, I'll fix that.

**Comment by ThomasWaldmann** [comment]
OK, I did fixes based on @rexfue's feedback and merge this now to not longer block other developments.

**Pull request merged**